# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/xcdroast/audio-dev.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff --unified --recursive xcdroast-0.98alpha9/io.c xcdroast-0.98alpha9_new/io.c
--- ./io.c.orig	Tue Jul 17 21:39:11 2001
+++ ./io.c	Thu Aug  2 17:47:24 2001
@@ -1004,11 +1004,11 @@
 #if defined(linux) || defined(__FreeBSD__)
 	/* for linux check if /dev/dsp or /dev/dsp1 exist */
 
-	if (stat("/dev/dsp",&buf) == 0) {
-		dsp = g_list_append(dsp,"/dev/dsp");
+	if (stat("/dev/sound/dsp",&buf) == 0) {
+		dsp = g_list_append(dsp,"/dev/sound/dsp");
 	}
-	if (stat("/dev/dsp1",&buf) == 0) {
-		dsp = g_list_append(dsp,"/dev/dsp1");
+	if (stat("/dev/sound/dsp1",&buf) == 0) {
+		dsp = g_list_append(dsp,"/dev/sound/dsp1");
 	}
 #endif
 #ifdef sun
@@ -1074,9 +1074,9 @@
 
 #if defined(linux) || defined(__FreeBSD__)
 
-	if (strncmp(dsp,"/dev/dsp",8) == 0) {
+	if (strncmp(dsp,"/dev/sound/dsp",14) == 0) {
 		strcpy(tmp,dsp+8);
-		g_snprintf(tmp2,MAXLINE,"/dev/mixer%s",tmp);
+		g_snprintf(tmp2,MAXLINE,"/dev/sound/mixer%s",tmp);
 
 		/* does device exist? */
 		if (stat(tmp2,&buf) == 0) {
diff --unified --recursive xcdroast-0.98alpha9/wavplay.c xcdroast-0.98alpha9_new/wavplay.c
--- ./wavplay.c.orig	Tue Jul 17 21:39:14 2001
+++ ./wavplay.c	Thu Aug  2 17:47:48 2001
@@ -49,7 +49,7 @@
 gchar *audiobuf;
 
 #if defined(linux) || defined(__FreeBSD__)
-#define DEFAULT_AUDIO_DEVICE "/dev/dsp"
+#define DEFAULT_AUDIO_DEVICE "/dev/sound/dsp"
 #elif defined (aix)
 #define DEFAULT_AUDIO_DEVICE ""
 #else

